src/fabric.c

```diff
@@ -389,6 +389,11 @@ void fi_ini(void)
 			" (default: no). Setting this to yes could improve"
 			" performance at the expense of making fork() potentially"
 			" unsafe");
+	fi_param_define(NULL, "universe_size", FI_PARAM_SIZE_T,
+			"Defines the maximum number of processes that will be"
+			" used by distribute OFI application. The provider uses"
+			" this to optimize resource allocations"
```

Comment:
I don't think we can say this unless all providers abide by this parameter, right?

Reply:
I think we need to define which providers the parameter works with as part of an update to some man page somewhere.  From Dmitry's comment, the problem is figuring out which man page should get this update.

As a general rule, it would be nice to move to general parameters that work with any provider, where it makes sense.

Reply:
Rather than try to keep documentation up to date, should we instead have callbacks that query parameters effectively register that provider as using that parameter?  I.e., if `fi_param_get_*()` took a provider name (or object), then `fi_info(1)` could display an actual list of which providers use which parameters.

(the above example might be a bit simplistic -- the providers might actually need to call `fi_param_get_*()` early enough in their initialization such that `fi_info(1)` can display the list of which providers queried which params, etc.)

Reply:
That's not a bad idea, but I don't easily see how to make it work.

Reply:
We might be able to make this work.  The fi_param_xxx() calls are public APIs, so to support DL providers, I think we need to drop any reference to 'provider_of_param'.  We could probably reuse the fi_param_define() call if we add a new fi_param_type, though coming up with a good type name is eluding me atm.